src/server/acl/helpers.cc

```diff
@@ -318,4 +318,26 @@ ParseAclSetUser<std::vector<std::string_view>&>(std::vector<std::string_view>&,
 
 template std::variant<User::UpdateRequest, ErrorReply> ParseAclSetUser<CmdArgList>(
     CmdArgList args, const CommandRegistry& registry, bool hashed, bool has_all_keys);
+
+std::string AclKeysToString(const AclKeys& keys) {
+  if (keys.all_keys) {
+    return "~*";
+  }
+  std::string result;
+  for (auto& glob : keys.key_globs) {
+    auto& [pattern, op] = glob;
```

Comment:
`for (auto& [pattern, op] : keys.key_globs)` 🤓 😅

Reply:
:rofl: